manifests/capacityscheduling/capacityscheduling_v1alpha1_elasticquota.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: elasticquotas.capacityscheduling.sigs.k8s.io
+spec:
+  group: capacityscheduling.sigs.k8s.io
```

Comment:
Group should be `scheduling.sigs.k8s.io`.

Reply:
Done